client/app/scripts/superdesk-search/views/search-results.html

```diff
@@ -16,24 +16,20 @@
 
             <div sd-multi-action-bar></div>
         </header>
-
-
-
-        <div class="content" sd-updown data-items="items._items" data-select="preview(item)">
-            <div class="loading-indicator" ng-show="loading" translate>loading</div>
-
-            <div sd-search-tags></div>
-
-            <div sd-shadow>
-                <ul class="{{view}}-view list-view">
-                    <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview}">
-                        <div sd-media-box></div>
-                    </li>
-                </ul>
+        <div class="content" sd-scrolled data-next-action="fetchNext" data-previous-action="fetchPrevious">
+            <div sd-updown data-items="items._items" data-select="preview(item)">
+                <div class="loading-indicator" ng-show="loading" translate>loading</div>
+                <div sd-search-tags></div>
+                <div>
+                    <ul class="{{view}}-view list-view">
+                        <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview}">
```

Comment:
Hi, its just a POC at the moment to see the scrolling loading the data. we don't know the specifics of how the list will be populated at this stage, so more specific scenarios we will handle later on.